src/app/contact/contact-form/contact-form.component.scss~

```diff
@@ -0,0 +1,136 @@
+.overlay {
```

Comment:
What's the reason for this? The Tilda at the end of the file name usually indicates a backup file. Is this a backup file?